core/txpool/txpool_test.go

```diff
@@ -123,7 +123,7 @@ func (bc *testBlockChain) SubscribeChainHeadEvent(ch chan<- core.ChainHeadEvent)
 }
 
 func (bc *testBlockChain) SenderCacher() *core.TxSenderCacher {
-	return core.NewTxSenderCacher(1)
+	return &core.TxSenderCacher{}
```

Comment:
This avoids starting goroutines.
Not a very nice way to shut down this without lots of textual modifications.

Reply:
Could you add a comment here explaining why it's ok to leave this uninitalized?

Reply:
actually I'll just fix this since it's a modification to our code to have 
`	SenderCacher() *core.TxSenderCacher`
as part of the `blockChain` interface in `txpool.go`